### PR TITLE
Add suggested content endpoint

### PR DESCRIPTION
**Pull request by psoleckimoj**
For adding in another endpoint for suggested content/you might like

- Add new endpoint

### CODE REVIEW

File: moj-be/modules/custom/moj_resources/config/install/rest.resource.you_might_like_resource.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+id: you_might_like_resource
```
Comment (romidane) [inline moj-be/modules/custom/moj_resources/config/install/rest.resource.you_might_like_resource.yml]: Can we rename it to suggestions? Across the board?

Comment (psoleckimoj) [reply moj-be/modules/custom/moj_resources/config/install/rest.resource.you_might_like_resource.yml]: Lol, yeah, I was going to ask suggest it

Comment (psoleckimoj) [reply moj-be/modules/custom/moj_resources/config/install/rest.resource.you_might_like_resource.yml]: @romidane sorted!

File: moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            404
+        );
+    }
+
+    protected function setLanguage()
+    {
+        return is_null($this->currentRequest->get('_lang')) ? 'en' : $this->currentRequest->get('_lang');
+    }
+
+    protected function setCategory()
```
Comment (romidane) [inline moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php]: probably want to set this to `setNodeId`

File: moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+            $container->get('request_stack')->getCurrentRequest(),
+            $container->get('language_manager')
+        );
+    }
+
+    public function get()
+    {
+        $suggestedContent = $this->suggestedContentApiClass->SuggestedContentApiEndpoint(
+            $this->paramater_language_tag,
+            $this->paramater_category,
```
Comment (romidane) [inline moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php]: `$this->parameter_node_id`?

File: moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+{
+    protected $suggestedContentApiController;
+
+    protected $currentRequest;
+
+    protected $availableLangs;
+
+    protected $languageManager;
+
+    protected $paramater_category;
```
Comment (romidane) [inline moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php]: `$paramater_node_id`?

Comment (psoleckimoj) [reply moj-be/modules/custom/moj_resources/src/Plugin/rest/resource/SuggestedContentResource.php]: I'll go through them all 👍

**Pull request merged**